app/modules/project/sagas.jsx

```diff
@@ -322,6 +322,40 @@ function* watchImportArduinoProject() {
   }
 }
 
+function* watchLoadConfigSchema() {
+  yield takeLatest(actions.LOAD_CONFIG_SCHEMA, function*() {
+    try {
+      const schema = yield call(apiFetchData, {
+        query: 'project.get_config_schema'
+      });
+      yield put(updateEntity(CONFIG_SCHEMA_KEY, schema));
+    } catch (e) {
+      console.error(e);
```

Comment:
Please raise here core.notifyError. Check other examples. If we have internal errors caused by PIO Core, we should who them to user UI and they will report to Github. See example https://github.com/platformio/platformio-home/issues/37